server/db/controllers/users.js

```diff
@@ -1,8 +1,6 @@
-import User from '../models/users';
```

Comment:
Is the `params` verified? For each method, I don't see any verification that the object contains all the fields you need it to have.